src/layout/MarkdownFormatter.jsx

```diff
@@ -370,38 +370,40 @@ export function MarkdownFormatter({ markdown, backgroundColor, dict }) {
           }
         },
         h2: ({ children }) => {
-          let headerText = children[0];
+          let headerText = React.Children.toArray(children).join("");
```

Comment:
```suggestion
```

As I was reviewing this in-depth, I think I found an easier fix than `React.Children` and all that. Here's what's interesting: whoever wrote this before expected `children` to be an array. That's why they wrote:
```
let headerText = children[0]
```
However, if you run this locally and console.log `children`, it's just a string. The easiest way to fix this is to rename `children` to `headerText` (as I do in the comments above and below) and just remove this line. 


Incidentally, this code worked in the past:
```
let headerText = children[0]
```
because a `String` type is really just an array of individual characters. Let's say `children` equaled `My Title`. `headerText` would then equal `M`. It works in JavaScript, but obviously was a bug.

Reply:
Thanks, have corrected